modules/pipeline/pipengine/actionexecutor/plugins/scheduler/executor/plugins/k8sjob/k8sjob.go

```diff
@@ -523,6 +523,13 @@ func (k *K8sJob) generateKubeJob(specObj interface{}, clusterInfo map[string]str
 		container.Command = append(container.Command, []string{"sh", "-c", job.Cmd}...)
 	}
 
+	// annotations
+	if len(job.TaskContainers) > 0 {
+		kubeJob.Spec.Template.Annotations = map[string]string{
+			apistructs.MSPTerminusDefineTag: job.TaskContainers[0].ContainerID,
```

Comment:
how about multiple containers

Reply:
ok, please add comments for it in the code.

Reply:
commented in `taskContainers` job definition